### DISCUSSION TITLE
Load files using wildcards

### DISCUSSION THREAD

**Discussion by binaryfire** [composer/composer#11504] [General]
Hi all

Any thoughts on adding support for loading files using wildcards? Eg:

```
"autoload": {
    "files": ["modules/**/helpers/*"]
}
```

This would really simplify things for modular codebases.

**Comment by Seldaek** [comment]
The list of files is hardcoded when the autoloader gets dumped though so newly added files would not be autoloaded until you dump again, and so it might be more confusing if we added this. 

As there is ongoing work to perhaps finally get function autoloading in PHP core, I think that'd be a better outcome if we had psr-4 style support for functions instead of auto-including hundreds of files.
